packages/core/lib/console.js

```diff
@@ -161,18 +161,29 @@ class Console extends EventEmitter {
     // errors/warnings in child process - specifically the error re: having
     // multiple config files
     const spawnOptions = { stdio: ["inherit", "inherit", "pipe"] };
+    let spawnInput;
+    if (options.config) {
```

Comment:
Do different kinds of falsiness matter here? What happens if you do `"config" in options` instead?

Just want to make sure we're covering our bases.

Reply:
Using `config` without the hashes doesn't do anything! So I think we're good unless you have any other scenarios in mind?

Reply:
Nah should be alright? Thanks!